mcptools/go.mod

```diff
@@ -0,0 +1,138 @@
+module github.com/FerretDB/FerretDB/mcptools
+
+go 1.24.4
```

Comment:
There should be a `toolchain` directive with the full version, and the `go` directive should omit the patch version, just like in other `go.mod` files